src/Ztm.WebApi/Controllers/TokenController.cs

```diff
@@ -0,0 +1,132 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using NBitcoin;
+using Ztm.Configuration;
+using Ztm.WebApi.Callbacks;
+using Ztm.WebApi.Models;
+using Ztm.WebApi.Watchers.TransactionConfirmation;
+using Ztm.Zcoin.NBitcoin.Exodus;
+using Ztm.Zcoin.Rpc;
+using Transaction = Ztm.WebApi.Models.Transaction;
+
+namespace Ztm.WebApi.Controllers
+{
+    [Route("api")]
+    [ApiController]
+    public class TokenController : ControllerBase
```

Comment:
I think we should make this controller as a dedicated to `issue-tokens` only (e.g. `IssueTokenController`).